Calculator/CalculatorLibrary/CalculatorLibrary.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Just for your information, in dotnet 6, we are more likely to use System.Text.Json

Newtonsoft has been around for ever and is good and full featured, but it comes from a 3rd party. System.Text.Json is from Microsoft, has less features but is rocket fdast

See:
Provides high-performance, low-allocating, and standards-compliant capabilities to process JavaScript Object Notation (JSON), which includes serializing objects to JSON text and deserializing JSON text to objects, with UTF-8 support built-in. It also provides types to read and write JSON text encoded as UTF-8, and to create an in-memory document object model (DOM) for random access of the JSON elements within a structured view of the data.

https://docs.microsoft.com/en-us/dotnet/api/system.text.json?view=net-5.0